### PR TITLE
Mayhem initial integration

### PR DESCRIPTION
**Pull request by code-terror**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Build Stage
```
Comment (xansec) [inline Dockerfile:1]: Please do not use git clone. Use `ADD` instead.

File: Mayhemfile [hunk 2]
```diff
@@ -0,0 +1,8 @@
+project: code-terror/rasn
```
Comment (xansec) [inline Mayhemfile:1]: Please use generic project name. Tests has been renamed to testsuite.

File: fuzzing/Cargo.toml [hunk 3]
```diff
@@ -13,4 +13,4 @@ publish = false
 rasn = { path = ".." }
```
Comment (xansec) [inline fuzzing/Cargo.toml:13]: I presume this is necessary.

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

File: fuzzing/Cargo.toml (hunk 3 above)
Comment (code-terror) [reply fuzzing/Cargo.toml:13]: yes. afl 0.8 breaks the build

**Comment by code-terror** [comment]
@xansec Done!

**Comment by xansec** [comment]
LGTM!

**Pull request merged**